### PR TITLE
refactor(explorer): explorer refactoring

### PR DESCRIPTION
**Pull request by pavelnikolov**
* refactor Explorer
* rename/remove fields
* add doc comments to GraphQL schema

task: none

### CODE REVIEW

File: explorer/graphql/app/schema.graphql [hunk 1]
```diff
@@ -230,24 +226,40 @@ type PageInfo {
...
   xblock(sourceChainID: BigInt!, height: BigInt!): XBlock
+
+  "Retrieve a range of XBlocks by source chain ID and height."
   xblockrange(from: BigInt!, to: BigInt!): [XBlock]!
+
+  "Get the total count of XBlocks."
   xblockcount: BigInt
+
+  "Retrieve a range of XMsgs by source chain ID and height."
   xmsgrange(from: BigInt!, to: BigInt!): [XMsg]!
```
Comment (kevinhalliday) [inline explorer/graphql/app/schema.graphql:243]: Do we use these? 

`xblockrange` in particular seems a little useless. Like - `from` and `to` what? we have many source chains. We'd need to at least add source chain (also comment isn't representative)

If we don't use any queries I say we remove

**Review by kevinhalliday — APPROVED** [review]

File: explorer/graphql/app/schema.graphql (hunk 1 above)
Comment (pavelnikolov) [reply explorer/graphql/app/schema.graphql:243]: XBlock has height so it's for example from height 5 to height 15. Regardless of source chain id. Does that make sense?

Good catch on the comment - I'll fix it.

Comment (pavelnikolov) [reply explorer/graphql/app/schema.graphql:243]: `xmsgrange` seems to be used by the UI.

Comment (idea404) [reply explorer/graphql/app/schema.graphql:243]: I belive this is used here https://github.com/omni-network/omni/blob/7e228e9a04cd9cc0aae326378d0444ca6c042f65/explorer/ui/app/routes/_index.tsx#L51

I think `to` and `from` are ID fields, so, you get the max count of xmsgs first, then `to` becomes that and `from` is `to` minus the number of messages you want to have

**Pull request merged**